### PR TITLE
Scenario 1003 - Institutional and Private

### PR DESCRIPTION
**Pull request by jyotich1996**
1. added controllers 
2. Replaced controllers not being used now

### CODE REVIEW

File: 1003-Student_logs_in_and_accesses_the_dashboard.jmx [hunk 1]
```diff
@@ -18,6 +18,10 @@
           <stringProp name="IncludeController.includepath">Get_all_the_User_Spaces.jmx</stringProp>
         </IncludeController>
         <hashTree/>
+        <IncludeController guiclass="IncludeControllerGui" testclass="IncludeController" testname="Get Global Entitlements of a user" enabled="true">
+          <stringProp name="IncludeController.includepath">D:\Office\Cup\GIT Repos\test-performance\Get_global_entitlements_of_a_user.jmx</stringProp>
```
Comment (piyush-mehta) [inline 1003-Student_logs_in_and_accesses_the_dashboard.jmx:22]: remove local path reference

File: 1003-Student_logs_in_and_accesses_the_dashboard.jmx [hunk 2]
```diff
@@ -26,12 +30,16 @@
           <stringProp name="IncludeController.includepath">Get_classes_of_an_user.jmx</stringProp>
         </IncludeController>
         <hashTree/>
-        <IncludeController guiclass="IncludeControllerGui" testclass="IncludeController" testname="Get list of all products for a user" enabled="true">
-          <stringProp name="IncludeController.includepath">Get_list_of_all_products_for_a_user.jmx</stringProp>
+        <IncludeController guiclass="IncludeControllerGui" testclass="IncludeController" testname="API to grant authkey on refid" enabled="true">
+          <stringProp name="IncludeController.includepath">D:\Office\Cup\GIT Repos\test-performance\API_to_grant_authkey_on_refid.jmx</stringProp>
```
Comment (piyush-mehta) [inline 1003-Student_logs_in_and_accesses_the_dashboard.jmx:34]: remove local path reference

File: 1003-Student_logs_in_and_accesses_the_dashboard_Private.jmx [hunk 3]
```diff
@@ -18,12 +18,20 @@
           <stringProp name="IncludeController.includepath">Get_all_the_User_Spaces.jmx</stringProp>
         </IncludeController>
         <hashTree/>
+        <IncludeController guiclass="IncludeControllerGui" testclass="IncludeController" testname="Get Global Entitlements of a user" enabled="true">
```
Comment (piyush-mehta) [inline 1003-Student_logs_in_and_accesses_the_dashboard_Private.jmx:21]: remove local path reference in this file also

File: 1003-Student_logs_in_and_accesses_the_dashboard.jmx (hunk 2 above)
Comment (jyotich1996) [reply 1003-Student_logs_in_and_accesses_the_dashboard.jmx]: Removed local path reference

File: 1003-Student_logs_in_and_accesses_the_dashboard.jmx (hunk 1 above)
Comment (jyotich1996) [reply 1003-Student_logs_in_and_accesses_the_dashboard.jmx]: Removed local path reference

**Pull request merged**